projects/ngrx.io/content/examples/testing-store/src/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <title>Angular App</title>
+  </head>
```

Comment:
Add `<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.99.0/jasmine.css">` to style the test page